routes/login.js

```diff
@@ -18,10 +18,10 @@ router.post('/signup', urlencodedParser, function(req, res){
 
 //Sign in logic
 router.post('/signin', urlencodedParser, async(req, res) => {
-    // check is id exist
+   console.log(req.body)
     let userid = await user.User.findOne({id: req.body.id});
     if (!userid) {
-        res.status(400).send('Invalid id please sign up');
+        res.status(300).send('Invalid id please sign up');
```

Comment:
use the right code for each use case
https://restfulapi.net/http-status-codes/
401 seems to be the right one in this case